Virtusize/Sources/UI/VirtusizeWebViewController.swift

```diff
@@ -145,6 +145,18 @@ public final class VirtusizeWebViewController: UIViewController {
 	@objc internal func shouldClose() {
 		dismiss(animated: true, completion: nil)
 	}
+
+    private func getWebViewURL() -> URLRequest? {
+        let storeId = product?.productCheckData?.storeId
+        if storeId == 99 {
```

Comment:
Magic number here. It should be defined in a constant file.
```swift
let storeData: [String: [String: String]] = [
    "UnitedArrow": [
        "storeId": "99"
    ],
    "leviKorea": [
        "storeId": "XXX"
    ]
]

```

Reply:
[0bc8a70](https://github.com/virtusize/integration_ios/pull/127/commits/0bc8a70cbe3c2a903148b65d9d7b4843d5a2019a)